scala/scala.bzl

```diff
@@ -661,18 +698,31 @@ _common_attrs = {
   "print_compile_time": attr.bool(default=False, mandatory=False),
 }
 
+library_attrs = {
+  "main_class": attr.string(),
+  "exports": attr.label_list(allow_files=False),
+}
+
+library_outputs = {
+  "jar": "%{name}.jar",
+  "deploy_jar": "%{name}_deploy.jar",
+  "ijar": "%{name}_ijar.jar",
+  "manifest": "%{name}_MANIFEST.MF",
+}
+
 scala_library = rule(
   implementation=_scala_library_impl,
   attrs={
-      "main_class": attr.string(),
-      "exports": attr.label_list(allow_files=False),
-      } + _implicit_deps + _common_attrs + _resolve_deps,
-  outputs={
-      "jar": "%{name}.jar",
-      "deploy_jar": "%{name}_deploy.jar",
-      "ijar": "%{name}_ijar.jar",
-      "manifest": "%{name}_MANIFEST.MF",
-      },
+      "enable_dependency_analyzer": attr.bool(default=True, mandatory=False),
+      "dependency_analyzer_plugin": attr.label(default=Label("@io_bazel_rules_scala//plugin/src/main:dependency_analyzer"), allow_files=_jar_filetype, mandatory=False),
```

Comment:
should this be a private attribute? are actual users supposed to be able to configure this? I think not so it should probably be internal (`_dependency_analyzer_plugin`)